src/utils/storage-utils.js

```diff
@@ -82,6 +81,10 @@ export function getStorageLocationTypeForPath(path) {
   return STORAGE_LOCATION_TYPE.unknown;
 }
 
+export function getMediaLibraryItemPath() {
```

Comment:
This would be `getMediaLibraryPath` actually 🤔  Or 
```
getMediaLibraryItemPath(itemId) {
     return `media-library/${itemId}`;
}
```

Reply:
Absolutely right, should be `getMediaLibraryPath` (there is actually no `itemId` in the path, just the filename, I think). Changed it.